package.json

```diff
@@ -686,10 +686,15 @@
           "description": "%java.debugger.launch.skipConstructors.description%",
           "default": false
         },
-        "java.debug.settings.limitOfVariablesPerJdwpRequest": {
+        "java.debug.settings.jdwp.limitOfVariablesPerJdwpRequest": {
           "type": "number",
-          "description": "%java.debugger.configuration.limitOfVariablesPerJdwpRequest.description%",
+          "description": "%java.debugger.configuration.jdwp.limitOfVariablesPerJdwpRequest.description%",
           "default": 100
+        },
+        "java.debug.settings.jdwp.requestTimeout": {
+          "type": "number",
+          "description": "%java.debugger.configuration.jdwp.requestTimeout.description%",
+          "default": 3000
```

Comment:
can limit min value here, e.g. 100 according to your implementation

Reply:
Nice finding.